src/languages/php.js

```diff
@@ -394,6 +393,42 @@ export default function(hljs) {
     ]
   };
 
+  const FUNCTION_INVOKE = {
+    relevance: 0,
+    match: [
+      /\b/,
+      // to prevent keywords from being confused as the function title
+      regex.concat("(?!fn\\b|function\\b|", normalizeKeywords(KWS).join("\\b|"), "|", normalizeKeywords(BUILT_INS).join("\\b|"), "\\b)"),
+      IDENT_RE,
+      regex.concat(WHITESPACE, "*"),
+      regex.lookahead(/(?=\()/)
+    ],
+    scope: {
+      3: "title.function.invoke",
+    }
+  };
+  FUNCTION_INVOKE.contains = [
```

Comment:
Why isn't this just inside the original declaration?

Reply:
Its self-reference FUNCTION_INVOKE

Reply:
I see, but just add the `contains` key along with all the others when you create the object literal.